src/app/modules/users/store/user.effects.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TestBed } from '@angular/core/testing';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { Action } from '@ngrx/store';
+import { Observable, of, throwError } from 'rxjs';
+import { UsersService } from '../services/users.service';
+import { UserActionTypes } from './user.actions';
+import { UserEffects } from './user.effects';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+
+describe('UserEffects', () => {
+  let actions$: Observable<Action>;
+  let effects: UserEffects;
+  let service: UsersService;
+  let toastrService;
+  const user = { id: 'id', name: 'name', email: 'email' };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [UserEffects, provideMockActions(() => actions$)],
+      imports: [HttpClientTestingModule, ToastrModule.forRoot()],
+      declarations: [],
+    });
+    effects = TestBed.inject(UserEffects);
+    service = TestBed.inject(UsersService);
+    toastrService = TestBed.inject(ToastrService);
+  });
+
+  it('should be created', async () => {
+    expect(effects).toBeTruthy();
+  });
+
+  it('should return a list of users when the action LOAD_USERS call succeeds', async () => {
+    actions$ = of({ type: UserActionTypes.LOAD_USERS });
+    const serviceSpy = spyOn(service, 'loadUsers');
+    serviceSpy.and.returnValue(of(user));
+
+    effects.loadUsers$.subscribe((action) => {
+      expect(action.type).toEqual(UserActionTypes.LOAD_USERS_SUCCESS);
+    });
+  });
+
+  it('should return an error when the action LOAD_USERS call fails', async () => {
```

Comment:
Wrong test description, you are not returning any error here. Instead, you're calling the error function from toastrService and you are expective a `LOAD_USER_FAIL` as action type

Reply:
I have changed the description as in the previous comment to have a standard for the test effects.